certbot-dns-rfc2136/certbot_dns_rfc2136/_internal/tests/dns_rfc2136_test.py

```diff
@@ -40,8 +40,19 @@ def setUp(self):
 
         self.mock_client = mock.MagicMock()
         # _get_rfc2136_client | pylint: disable=protected-access
+        self.orig_get_client = self.auth._get_rfc2136_client
         self.auth._get_rfc2136_client = mock.MagicMock(return_value=self.mock_client)
 
+    def test_get_client(self):
```

Comment:
```suggestion
    def test_get_client_default_conf_values(self):
```
Given the comment, perhaps a more descriptive name then?